pytorch_to_returnn/torch/nn/modules/module.py

```diff
@@ -773,7 +778,7 @@ def _base_get_output_shape_from_returnn(inputs_flat: List[Tensor], layer: LayerB
       if x.returnn_data.have_batch_axis():
         batch_size = input.shape[x.returnn_axis_from_torch_axis[x.returnn_data.batch_dim_axis]]
       for i in x.returnn_data.get_dynamic_axes():
-        dim_tag = x.returnn_data.get_dim_tag(i)
+        dim_tag = _get_unique_spatial_dim_tag_name(x.returnn_data, i)
```

Comment:
I'm not sure if `dim_tag_name` or so is maybe better.

But in any case, you need to fix the documentation of `dyn_size_dim_tag_to_spatial_idx_and_torch_dim`. Now it's not anymore "RETURNN dim tag" but "RETURNN unique dim tag name" or so.